controllers/warehouse-controller.js

```diff
@@ -9,6 +9,28 @@ const index = async (_req, res) => {
   }
 };
 
+//this is to find a single warehouse
+const findWarehouse = async (req, res) => {
+  try {
+    const warehousesFound = await knex("warehouses").where({
+      id: req.params.id,
+    });
+    if (warehousesFound.length === 0) {
+      return res.status(404).json({
+        message: `warehouse with ID ${req.params.id} not found`,
+      });
+    }
+    const warehouseData = warehousesFound[0];
+    res.json(warehouseData);
+    res.status(200).json(data);
```

Comment:
should line 24 be: res.status(200).json(warehouseData) and line 25 is redundant?

Reply:
Agreed

Reply:
Hi guys. Good catch. I will change that. Thank you.

Reply:
Hi all, I just made a new change for lines 24 and 25. You may check the changes at the Files Changed tab on top and check line 24. So this would be ready for merge. thank you

Reply:
Thanks Elizabeth